tests/stt/checksExecution_test.js

```diff
@@ -23,10 +23,16 @@ const cleanup = async () => {
 Feature('Security Checks: Checks Execution');
 
 BeforeSuite(async ({ perconaServerDB, addInstanceAPI }) => {
+  const mysqlComposeConnection = {
+    host: '127.0.0.1',
```

Comment:
doesn't it work with `host: mysql` ?

Reply:
no it doesn't the DB helper is running on the worker node, doesn't really see the same network as the server and mysql db compose.